benches/PERFORMANCE.json

```diff
@@ -3792,255 +3874,667 @@
                     "common_prefix": "air::farewell_step::outcome",
                     "duration": "290.00µs",
                     "nested": {
-                      "populate_outcome_from_contexts": "236.00µs"
+                      "populate_outcome_from_contexts": "235.00µs"
                     }
                   }
                 }
               },
-              "signing_step::sign_produced_cids": "95.00µs",
+              "signing_step::sign_produced_cids": "99.00µs",
               "verification_step::verify": {
                 "common_prefix": "air_interpreter_data::cid_info",
-                "duration": "435.00µs",
+                "duration": "426.00µs",
                 "nested": {
-                  "verify": "93.00µs"
+                  "verify": "75.00µs"
                 }
               }
             }
           }
         },
-        "total_time": "2.71ms"
+        "total_time": "2.74ms"
       }
     },
-    "datetime": "2023-11-06 18:49:22.923773+00:00",
+    "datetime": "2023-11-27 15:04:01.881477+00:00",
     "features": "check_signatures,gen_signatures",
-    "platform": "macOS-14.0-arm64-arm-64bit",
-    "version": "0.53.0"
+    "platform": "macOS-14.1.1-arm64-arm-64bit",
+    "version": "0.54.0"
   },
   "e536f8eaae8c978493a773ba566ae3393e2e6240d6ea8e05b5ca1b8f77e9c441": {
     "benches": {
-      "big_values_data": {
+      "big-values-data": {
         "comment": "Loading a trace with huge values",
         "memory_sizes": [
-          "58.938 MiB",
-          "58.938 MiB"
+          "59.000 MiB",
+          "59.000 MiB"
         ],
         "stats": {
           "air::runner::execute_air": {
             "common_prefix": "air",
-            "duration": "12.11ms",
+            "duration": "17.72ms",
```

Comment:
I am curious if this is a degradation and something we expect?

Reply:
Should be signatures processing overhead?

Reply:
CID verification.